srcts/extras/globalShiny.ts

```diff
@@ -2,19 +2,16 @@
 // Project: Shiny <https://shiny.rstudio.com/>
 // Definitions by: RStudio <https://www.rstudio.com/>
 
-import type { Shiny as RStudioShiny } from "../src/shiny/index";
+import type { Shiny as ShinyClass } from "../src/shiny/index";
 
 declare global {
   // Tell Shiny variable globally exists
   // eslint-disable-next-line @typescript-eslint/naming-convention
-  const Shiny: RStudioShiny;
+  const Shiny: ShinyClass;
 
   // Tell window.Shiny exists
   interface Window {
     // eslint-disable-next-line @typescript-eslint/naming-convention
-    Shiny: RStudioShiny;
+    Shiny: ShinyClass;
   }
-
-  // Make `Shiny` a globally available type definition. (No need to import the type)
-  type Shiny = RStudioShiny;
```

Comment:
This type didn't seem to be used anywhere, and it was confusing that the type and the variable had the same name.

Reply:
Should check if bslib complains about this change.